### PR TITLE
Added description to README.md file

### PR DESCRIPTION
**Pull request by MinoDentori**
1. Please provide feedback to the parts, of the text, that don't look like human written, cause I used GPT a lot.
2. Need to complete my Postman collection, before inserting it into this file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,99 @@
...
+### Cart
+
++ _`GET: /api/cart` - endpoint for getting all items in your shopping cart (User authority)_
++ _`POST: /api/cart` - endpoint for adding items in your shopping cart (User authority)_
++ _`PUT: /api/cart/cart-items/{itemId}` - endpoint for updating items quantity (User authority)_
++ _`DELETE: /api/cart/cart-items/{itemId}` - endpoint for deleting items from your shopping cart (User authority)_
+
+
+## Setup
+
```
Comment (d1sam) [inline README.md:75]: what about docker-compose? also please provide maven commands.

File: README.md [hunk 2]
```diff
@@ -1 +1,99 @@
...
+Clone the repository to your local machine.
+Ensure you have Java, Maven, and your preferred IDE installed.
+Configure the database settings in the application.properties file.
+Run the application using your IDE or the command line.
+Access the platform through your web browser.
+
+## Challenges and Solutions
+
+The journey of creating Bookstore Pro wasn't without its challenges:
```
Comment (d1sam) [inline README.md]: please rewrite this section without GPT and describe deeper challenges, that you overcomed.

File: README.md
Comment (d1sam) [inline README.md:1]: please also add Possible improvements section.

**Review by d1sam — CHANGES_REQUESTED** [review]
Good job, but there are some improvements should be done!

File: README.md
Comment (MinoDentori) [reply README.md:1]: fixed

**Review by d1sam — APPROVED** [review]
well done!

**Pull request merged**